frontend/app/src/components/dashboard/DashboardAssetTable.vue

```diff
@@ -124,76 +134,132 @@ import {
       },
       {
-        text: i18n.t('dashboard_asset_table.headers.price').toString(),
+        text: i18n
+          .t('dashboard_asset_table.headers.price', {
+            symbol: currencySymbol.value
+          })
+          .toString(),
         value: 'usdPrice',
-        align: 'end'
+        align: 'end',
+        class: 'text-no-wrap'
       },
       {
-        text: i18n.t('dashboard_asset_table.headers.amount'),
+        text: i18n.t('dashboard_asset_table.headers.amount').toString(),
         value: 'amount',
         align: 'end',
         cellClass: 'asset-divider'
       },
       {
-        text: i18n.t('dashboard_asset_table.headers.value').toString(),
+        text: i18n
+          .t('dashboard_asset_table.headers.value', {
+            symbol: currencySymbol.value
+          })
+          .toString(),
         value: 'usdValue',
         align: 'end',
         class: 'text-no-wrap'
-      },
-      {
+      }
+    ];
+
+    if (visibleColumns.includes(TableColumn.PERCENTAGE_OF_TOTAL_NET_VALUE)) {
+      headers.push({
         text: totalNetWorthUsd.value.gt(0)
-          ? i18n.t('dashboard_asset_table.headers.percentage').toString()
+          ? i18n
+              .t('dashboard_asset_table.headers.percentage_of_total_net_value')
+              .toString()
           : i18n.t('dashboard_asset_table.headers.percentage_total').toString(),
-        value: 'percentage',
+        value: 'percentageOfTotalNetValue',
         align: 'end',
         cellClass: 'asset-percentage',
         class: 'text-no-wrap',
         sortable: false
-      }
-    ];
+      });
+    }
+
+    if (
+      visibleColumns.includes(TableColumn.PERCENTAGE_OF_TOTAL_CURRENT_GROUP)
+    ) {
+      headers.push({
+        text: i18n
+          .t(
+            'dashboard_asset_table.headers.percentage_of_total_current_group',
+            {
+              group: title.value
+            }
+          )
+          .toString(),
+        value: 'percentageOfTotalCurrentGroup',
+        align: 'end',
+        cellClass: 'asset-percentage',
+        class: 'text-no-wrap',
+        sortable: false
+      });
+    }
+
+    return headers;
   });
 
 const DashboardAssetTable = defineComponent({
   name: 'DashboardAssetTable',
+  components: { VisibleColumnsSelector, MenuTooltipButton },
   props: {
     loading: { required: false, type: Boolean, default: false },
     title: { required: true, type: String },
     balances: {
       required: true,
       type: Array as PropType<AssetBalanceWithPrice[]>
-    }
+    },
+    tableType: { required: true, type: String as PropType<DashboardTableType> }
   },
   setup(props) {
-    const { balances } = toRefs(props);
+    const { balances, title, tableType } = toRefs(props);
     const search = ref('');
 
     const currencySymbol = currency;
     const exchangeRate = setupExchangeRateGetter();
+    const totalInUsd = computed(() => {
+      return aggregateTotal(balances.value, 'USD', new BigNumber(1));
+    });
```

Comment:
Is there a difference between the `totalInUsd` property and the `totalUsd` property on line 283?

Reply:
it is same, removed now

Reply:
There doesn't seem to be any difference between the two. If that is indeed the case would it be easy to remove this, and replace its usages with `totalUsd` instead?

Reply:
I used the highlighted code instead, instead use the code below that which do calculation on the components.